modelci/persistence/service.py

```diff
@@ -16,7 +16,7 @@ class ModelService(object):
     @classmethod
```

Comment:
will this old service.py will be removed later?

Reply:
yes